### ISSUE TITLE
Issues about figures.

### ISSUE THREAD

**Issue by KennyNH** [Jhryu30/AnomalyBERT#2]
Thanks for your wonderful work!

This work focuses on multivariate time series data such SWaT, WADI and SMAP. But the curves of 'Abnormal Component' in Figure 1, 4 and 5 only show the univariate time series. How do you select the dimension? Or do you do conduct some operations to unify all the dimensions?

**Comment by coffeetumbler** [comment]
Hello, @KennyNH.

In our figures, the "Abnormal Component" label refers to columns whose values are affected by outliers. We manually chose one anomalous column for each example window to show anomalous behavior in the original data.

For example, in Figure 4, an outlier occurs in the second column of the SWaT dataset ('LIT101' in raw data). We visualize this column as an abnormal component.

For the SMAP dataset, only the first column contains numerical values (not categorical values). We observed that all outliers in SMAP affect this column. (That is, the abnormal component for SMAP is the first column.)
